extensions/amp-story-interactive/0.1/amp-story-interactive.css

```diff
@@ -110,3 +110,94 @@
+.i-amphtml-story-interactive-container:not(.i-amphtml-story-interactive-active) .i-amphtml-story-interactive-disclaimer {
+  display: none !important;  /* Disable the disclaimer if not on active page to prevent tabbing */
+}
+
+.i-amphtml-story-interactive-disclaimer-alert {
+  width: 1.75em !important;
+  height: 1.75em !important;
+  position: absolute !important;
+  right: 0 !important;
+  bottom: 0 !important;
+  transform-origin: bottom right !important;
+  pointer-events: all !important;
+  background-color: var(--interactive-accent-color) !important;
+  border: none !important;
+  border-radius: 50% !important;
+  box-shadow: 0 .2em .2em 0 #00000040 !important;
+  padding: .25em !important;
+  cursor: pointer !important;
+  font-size: inherit !important;
+}
+
+.i-amphtml-story-interactive-disclaimer[active]
+  .i-amphtml-story-interactive-disclaimer-alert {
+  display: none !important;
+}
+
+.i-amphtml-story-interactive-disclaimer-alert::after {
+  content: '';
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="4" height="12" viewBox="0 0 4 12" fill="black"><path d="M2.00002 0C1.8191 2.71375e-07 1.64002 0.0361432 1.47343 0.10628C1.30683 0.176418 1.15613 0.279119 1.03026 0.408281C0.904391 0.537443 0.805932 0.690432 0.740732 0.858156C0.675531 1.02588 0.644921 1.20492 0.650718 1.38463L0.921037 6.88156C0.930028 7.15999 1.04765 7.42404 1.24901 7.61784C1.45038 7.81164 1.7197 7.92 2.00002 7.92C2.28034 7.92 2.54966 7.81164 2.75103 7.61784C2.9524 7.42404 3.07001 7.15999 3.07901 6.88156L3.34933 1.38463C3.35513 1.20492 3.32452 1.02588 3.25932 0.858158C3.19412 0.690434 3.09566 0.537445 2.96979 0.408282C2.84392 0.279119 2.69322 0.176418 2.52662 0.10628C2.36003 0.036143 2.18095 2.57507e-07 2.00002 0Z"/><path d="M2 9.30005C1.73299 9.30005 1.47199 9.37924 1.24999 9.52758C1.02798 9.67592 0.854954 9.88676 0.75278 10.1334C0.650606 10.3801 0.623875 10.6516 0.675968 10.9134C0.728061 11.1753 0.856639 11.4159 1.04544 11.6047C1.23424 11.7935 1.47479 11.922 1.73666 11.9741C1.99854 12.0262 2.26997 11.9995 2.51665 11.8973C2.76333 11.7951 2.97417 11.6221 3.12251 11.4001C3.27085 11.1781 3.35002 10.917 3.35002 10.65C3.35002 10.4728 3.3151 10.2972 3.24726 10.1334C3.17941 9.96963 3.07997 9.8208 2.95461 9.69544C2.82925 9.57008 2.68042 9.47064 2.51663 9.4028C2.35283 9.33496 2.17728 9.30004 2 9.30005Z"/></svg>') !important;
+  background-repeat: no-repeat !important;
+  background-position: center !important;
+  background-size: 0.25em !important;
+  background-color: white !important;
+  border-radius: 50% !important;
+  width: 1.25em !important;
+  height: 1.25em !important;
+  display: block !important;
+}
+
+.i-amphtml-story-interactive-disclaimer-bubble {
+  overflow: hidden !important;
+  padding: 1em !important;
+  text-align: initial !important;
+  box-sizing: border-box !important;
+  font-size: 1em !important;
+  background: white;
+  border-radius: 1em;
+  box-shadow: 0px .2em .2em 0px #00000040 !important;
```

Comment:
`#00000040` could be a top level var since it's used more than once.
It might seem like overkill but colors can get out of hand very fast.

Reply:
Also moved the blue link color to a variable.